pillar.example

```diff
@@ -1,6 +1,10 @@
 vault:
-  lookup:
-    pkg: myvault
-    conf_file: /usr/local/etc/vault/vault.conf
-  config:
-    key: othervalue
+  overrides:
+    keybase_users:
+      - renaissancedev
+      - pdpinch
+      - bdero
+    secret_shares: 3
+    secret_threshold: 2
+
+vault.verify: False
```

Comment:
Is vault.verify used for something? I can't find it referenced in the rest of this PR.
Also, I noticed there's a grain defined for this. Do we need `vault.verify` to be both pillar data and a grain?